specification/cosmos-db/resource-manager/Microsoft.DocumentDB/stable/2024-05-15/managedCassandra.json

```diff
@@ -946,6 +946,23 @@
             "provisionError": {
               "$ref": "#/definitions/CassandraError",
               "description": "Error related to resource provisioning."
+            },
+            "azureConnectionMethod": {
+              "type": "string",
+              "description": "How to connect to the azure servcies needed for running the cluster",
+              "enum": [
+                "None",
+                "VPN"
+              ],
+              "x-ms-enum": {
+                "name": "AzureConnectionType",
+                "modelAsString": true
+              }
+            },
+            "privateLinkResourceId": {
+              "type": "string",
+              "readOnly": true,
+              "description": "If the Connection Method is Vpn, this is the Id of the private link resource that the datacenters need to connect to."
             }
```

Comment:
format : "arm-id".

[https://github.com/Azure/autorest/blob/main/docs/extensions/readme.md#x-ms-arm-id-details](https://github.com/Azure/autorest/blob/main/docs/extensions/readme.md#x-ms-arm-id-details)

Reply:
dont you need to include the allowed resources to restrict it to privae links ?

Reply:
@guanzhousongmicrosoft @DrewKaufmann can you take a look at this suggestion as well?

Reply:
Added by @DrewKaufmann. Resolving conversation.